acceptance/tests/connect/connect_inject_test.go

```diff
@@ -208,8 +213,13 @@ func TestConnectInject_MultiportServices(t *testing.T) {
 			}
 
 			logger.Log(t, "creating multiport static-server and static-client deployments")
-			k8s.DeployKustomize(t, ctx.KubectlOptions(t), cfg.NoCleanupOnFailure, cfg.NoCleanup, cfg.DebugDirectory, "../fixtures/bases/multiport-app")
-			k8s.DeployKustomize(t, ctx.KubectlOptions(t), cfg.NoCleanupOnFailure, cfg.NoCleanup, cfg.DebugDirectory, "../fixtures/cases/static-client-inject-multiport")
+			if cfg.EnableWindows {
```

Comment:
have kept these here. since this is something we need to support.  but  test is currently not working.
the ones which are not needed have been skipped and no windows related change are kept

Reply:
May be add a todo at the place you skipped the test.

Reply:
added. https://github.com/hashicorp/consul-k8s/pull/2703/commits/1bdbf2533357ab5342aa0e0833329643c8f5613e